pkgs/t3/default.nix

```diff
@@ -0,0 +1,29 @@
+{
+  help2man,
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "t3";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "flox";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-NB1aNnMnFhv5rvTvml7vLSJlH94OW9zplttk8HrAy+8=";
+  };
+
+  installFlags = [ "PREFIX=$(out)" ];
+  nativeBuildInputs = [ help2man ];
+  doCheck = true;
+
+  meta = with lib; {
+    homepage = "https://github.com/flox/t3";
+    description = "Next generation tee with colorized output streams and precise time stamping";
+    maintainers = [ maintainers.limeytexan ];
+    license = licenses.mit;
+  };
+}
```

Comment:
suggestion nonblocking: can we enable branch protection and get CI running tests on the t3 repo?

Reply:
Should we be running the `nix build` in CI then?

Reply:
We are doing that already by way of the flox build that pulls in the t3 package on each of the supported architectures so I don't think necessary, particularly as we should all expect t3 to change very infrequently.

I've turned on branch protection for t3 now that it's at the point of seeing production use. Resolving conversation - please reply if I've missed anything!